governance/sentinel-cidr-whitelist/README.md

```diff
@@ -0,0 +1,106 @@
+#  Vault Sentinel policy
+This guide shows how to enable a Sentinel policy in Vault that will only allow access to a secret enpoint if the request comes from a certain ip address.
+
+Sentinel policies can be bound to any path within Vault using the *egp* endpoint, or to any Identity entities and groups or to tokens using the *rgp* endpoint. Additional information can be found [here][sentinel-docs].
+
+This guide only uses Sentinel for IP whitelisting of a secret endpoint. However there are many other types of checks available for Sentinel, such as request time, token attributes, path attributes and more. A complete list can be found [here][sentinel-properties]
+
+## Enterprise only
+Please note that Sentinel is only available in the Enterprise version of Vault
+
+## Estimated Time to Complete
+This exercise should only take 5-10 minutes to complete for a user familiar with Linux.
+
+## Steps to deploy and test Sentinel policy
+
+## Files in this repository
+
+File | Description | Application
+--- | --- | ---
+egp-payload.json | Contains base64 enconded Sentinel policy | Used to register the Sentinel policy within Vault
+secret-example.json | Value representing sensitive information to be stored in Vault | Used to test if user can read/write a secret when Sentinel policy is in place
+cidr-policy.sentinel | The checks this Sentinel policy will enforce | Needs to be converted to base64 prior to be added to *egp-payload.json*
+my-acl-policy.json | ACL policy to be associated to an user | This policy will allow access to the secret path that will be protected by Sentinel
+user-password.json | Contains user password | Used to create a new user with the permissive ACL policy
+user-payload | Information to create an user | This user will test the Sentinel check
+
+### Optional
+The Sentinel policy needs to be encoded as a base64 string prior to submitting to Vault. In this repository the Sentinel policy "cidr-policy.sentinel" is already encoded, however if you would like to change or use your own you can use a service such as https://www.base64decode.org/ to encode/decode a string.
+
```

Comment:
Maybe show a bash example of base64 encoding?
Terraform example with built in function ?https://www.terraform.io/docs/configuration/interpolation.html#base64encode-string-